trin-history/src/events.rs

```diff
@@ -29,12 +29,12 @@ impl HistoryEvents {
                 }
                 Err(e) => {
                     error!("failed to process portal history event: {}", e);
-                    e.into_bytes()
+                    e.to_string().into_bytes()
```

Comment:
Yeah, I agree. Initially I changed everything to be `Err(error)` and not `Err(e)`, but I think I switched back to avoid unrelated changes. Looks like I missed some of those. I'll go ahead and update the `Err(e)` to `Err(error)` in parts of the code that this PR touches. I agree that we should standardize a convention in a later PR.